src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -1,143 +1,61 @@
 
-image::Koji-screenshot=1.svg[Render a view for the rich preview image]
+image::Koji-screenshot=1.png[Render a view for the thumbnail image]
 
-For the following example, assume the template supports remixes of the title, font size, and logo.
+For the following example, assume the app supports customizing the image defined by `customImage`.
 
-. In your application code, add a conditional statement to display a different image for social shares.
+. In your application code, add a conditional statement to display a custom image for social shares.
 For example:
 +
-[.tabs,scope="lang"]
---
-.React
-====
-[.intro]
-Add a conditional statement in the `render` function.
-
-[source,JavaScript]
-----
-render() {
-  if (window.location.search.includes('koji-screenshot=1')) {
-    return (
-      <SocialSharePreview>
-          <H1
-          fontSize={this.state.titleOptions.fontSize}
-          >{this.state.titleOptions.title}</H1>
-          <Image src={this.optimizeURL(this.state.logo)} />
-        </SocialSharePreview>
-    );
-  }
-  ...
-}
-----
-====
-
-.Vanilla JS
-====
-[.intro]
-Wrap a conditional statement around the `render` function.
-
-[source,JavaScript]
-----
-// render the social share preview
-const socialShareRender = () => {
-    document.body.innerHTML = `
-        <div id="socialSharePreview">
-            <h1 style="font-size:${titleSize}px" id="title">${title}</h1>
-            <img id="logo" src="${optimizeURL(logo)}"/>
-        </div>`;
-}
-
-// render
-if (window.location.search.includes('koji-screenshot=1')) {
-    socialShareRender();
-} else {
-    render();
-}
-----
-====
-
-.Angular
-====
-[.intro]
-Add a conditional statement in the `ngOnInit` function and use it to apply a dynamic `id`.
-
-[.tabs]
-=====
-.app.component.ts
 [source,JavaScript]
 ----
-ngOnInit() {
-  ...
-  this.social = false
-  if (window.location.search.includes('koji-screenshot=1')) {
-    this.social = true
-  }
-  ...
+...
+if (Koji.playerState.context === 'screenshot') {
+  return (
+    <SocialSharePreview>
+      <h1>This is a custom thumbnail.</h1>
+      <img src={Koji.remix.get().customImage} />
+    </SocialSharePreview>
+  );
 }
-----
 
-.app.component.html
-[source,JavaScript]
-----
-<div class="container" id="{{social ? socialSharePreview : ''}}">
-        <h1 style="fontSize: {{titleOptions.fontSize}}px;">{{titleOptions.title}}</h1>
-        <img src="{{optimizeURL(logo)}}"
-    />
-    </div>
-----
-=====
-====
-
-.Svelte
-====
-[.intro]
-Add a conditional statement in the `App.svelte` file.
-
-[source,JavaScript]
-----
-{#if (window.location.search.includes('koji-screenshot=1'))}
-    <div class="social-share" style="background-image: url('{optimizeURL(image)}');">
-    <h1 style='font-size: {titleOptions.fontSize}px'>{titleOptions.title}</h1>
-    </div>
-{:else}
+// If the context is not equal to 'screenshot', return the HTML for the preview.
```

Comment:
"Preview" is a bit confusing here. The default view, that is, the running app, would be the "else" clause. Also, apps will have views for other contexts. In particular, this example app must render a view for the `remix` context, since it allows creators to customize values. We don't necessarily need to show those conditions in this code sample, since they are not directly related to this page. Maybe just update this part to a clarifying comment.